hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.tsx

```diff
@@ -77,6 +78,22 @@ describe("<PendingTables />", () => {
         expect(getByText(/show all/i)).toBeInTheDocument();
         expect(queryByText(/cancel order/i)).toBeNull();
     });
```

Comment:
Add the tests for this case : when "cancel" button is clicked, you need to make sure that the modal is also closed

Reply:
done